manual/cypher/cypher-docs/src/docs/dev/ql/functions/index.asciidoc

```diff
@@ -283,3 +283,24 @@ include::split.asciidoc[leveloffset=+1]
 include::reverse.asciidoc[leveloffset=+1]
 
 include::tostring.asciidoc[leveloffset=+1]
+
+ifndef::asciidoctor[:leveloffset: 1]
+
+[[query-functions-udf]]
+== User-defined functions ==
+User-defined functions are written in Java, deployed into the database and are called in the same way as any other Cypher function.
```

Comment:
This doesn't detail _how_ they are deployed into the database. Perhaps referring to procedures would be enough?

... deployed into the database similarly to procedures (_link to procedure section_), and ...